pkgs/applications/misc/rsclock/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rsClock";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "valebes";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fpidswkgpf9yr4vxqn38livz6r3z5i0lhg7ngj9f1ki4yqxn9zh";
+  };
+
+  doCheck = false;    
```

Comment:
I'm really sorry, I got confused. ``doCheck`` is false by default so this can be removed.

Reply:
No problems! :-)